packages/react-components/src/components/AppFrame/AppFrame.tsx

```diff
@@ -30,30 +32,39 @@ const Frame = (props: IAppFrameProps) => {
     isVisible: isSideNavigationVisible,
     elementRef: sideNavWrapperRef,
   });
+  const { isMobile, handleResize } = useMobileViewDetector({
```

Comment:
To avoid confusing the name `handleResize` with a function, I'd suggest to add it for example: `handleResizeRef`